assets/src/application/ui/display/TimezoneTimeInfo/Trigger.tsx

```diff
@@ -2,23 +2,28 @@ import React, { forwardRef } from 'react';
 
 import { GlobalOutlined } from '@appDisplay/icons/svgs';
 import { IconButton } from '@application/ui/input';
+import { useMemoStringify } from '@application/services/hooks';
+import { TooltipProps } from '@infraUI/display';
 
 interface TriggerProps {
 	tooltip: string;
 	onClick?: VoidFunction;
 }
 
-const Trigger = forwardRef<typeof IconButton, TriggerProps>(({ tooltip, ...props }, ref) => (
-	<IconButton
-		{...props}
-		borderless
-		color='white'
-		className='ee-timezone-info__button'
-		icon={() => <GlobalOutlined noMargin size='small' />}
-		tooltip={tooltip}
-		tooltipProps={{ placement: 'top' }}
-		ref={ref}
-	/>
-));
+const Trigger = forwardRef<typeof IconButton, TriggerProps>(({ tooltip, ...props }, ref) => {
+	const tooltipProps = useMemoStringify<TooltipProps>({ placement: 'top' });
+	return (
+		<IconButton
+			{...props}
+			borderless
+			color='white'
+			className='ee-timezone-info__button'
+			icon={() => <GlobalOutlined noMargin size='small' />}
```

Comment:
what about callbacks like this?
they can be wrapped in `useCallback` no?

we can do that in another PR though

Reply:
or even better we might update `IconButton` such that it receives just JSX;
I think I read in one of KCD articles that passing JSX from the parent is more performant than rendering that at child level

Reply:
No they cannot be wrapped in `useCallback` because `icon` expects a React Component. May be `React.memo` instead.